lib/app/core/data/model/user_data.dart

```diff
@@ -0,0 +1,55 @@
+class UserData {
+  UserData({
+     this.message,
+     this.data,
+  });
+
+  String? message;
+  Data ?data;
```

Comment:
rename your Data class UserProfile or something that is more relevant to the properties encapsulated in the class.

Reply:
changes done.